guava/src/com/google/common/util/concurrent/ServiceManager.java

```diff
@@ -55,10 +55,7 @@
 import com.google.j2objc.annotations.WeakOuter;
 import java.lang.ref.WeakReference;
 import java.time.Duration;
-import java.util.Collections;
-import java.util.EnumSet;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Dont use .* based imports

Reply:
it seems intellIj was set to automatically optimize the imports. changed it now.